dlopen the sonamed libs directly, like it has already been done for libGL.so.1,
as we don't want to Depend on the -devel packages for the .so files:
- libOSMesa.so.8: Mesa-libOSMesa-9.2.0-2
- libGLESv2.so.2: Mesa-libGLES-9.2.0-2
- libEGL.so.1: Mesa-libEGL-9.2.0-2
--- chromium-browser-36.0.1985.143/ui/gl/gl_implementation_osmesa.cc~	2014-08-12 22:02:27.000000000 +0300
+++ chromium-browser-36.0.1985.143/ui/gl/gl_implementation_osmesa.cc	2014-08-18 23:09:25.208179075 +0300
@@ -37,7 +37,7 @@
     return false;
   }
 
-  base::FilePath library_path = module_path.Append("libosmesa.so");
+  base::FilePath library_path = module_path.Append("libOSMesa.so.8");
   base::NativeLibrary library = LoadLibraryAndPrintError(library_path);
   if (!library)
     return false;
 
